containers/Ideas/ExplorePage.js

```diff
@@ -1,33 +1,67 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useCallback } from 'react'
+import { CommonActions } from '@react-navigation/native'
 import { View, FlatList } from 'react-native'
+import { useFocusEffect } from '@react-navigation/native'
 import API from '../../BaseApi'
 import { ExploreListItem } from './ExploreListItem'
 import { styles } from './ExplorePage.styles'
 import { ButtonSignOut } from '../../components/ButtonSignOut'
 import { TextSourceSans } from '../../components/TextSourceSans'
+import { useAuthorization } from '../Auth/AuthProvider.js'
 
 export const ExplorePage = (props) => {
   const [projects, setProjects] = useState([])
+  const { deepLink, setDeepLink } = useAuthorization()
 
   const fetchProjects = () => {
     API.getProjects()
       .then(response => response && setProjects(response))
       .catch(error => console.warn(error))
   }
 
+  const fetchProject = () => {
+    return API.getProject(deepLink.organisation, deepLink.project).then((result) => {
+      if (result.length) {
+        return result[0]
+      }
+      return null
+    })
+  }
+
   const pressHandler = (project) =>
     props.navigation.navigate('IdeaProject', { project: project })
 
   const projectItem = ({ item }) => (
     <ExploreListItem item={item} action={(project) => pressHandler(project)} />
   )
 
-  useEffect(() => {
-    const projectsListener = props.navigation.addListener('focus', () => {
-      fetchProjects()
-    })
-    return projectsListener
-  }, [])
+  useFocusEffect(
+    useCallback(() => {
+      if (deepLink.organisation !== null){
```

Comment:
on startup of the app i am receiving an error deepLink is null.

Reply:
arg :facepalm: fixed